whipper/command/cd.py

```diff
@@ -476,18 +484,34 @@ def _ripIfNotRipped(number):
                     tries -= 1
                     logger.critical('giving up on track %d after %d times',
                                     number, tries)
-                    raise RuntimeError("track can't be ripped. "
-                                       "Rip attempts number is equal to %d",
-                                       self.options.max_retries)
-                if trackResult.testcrc == trackResult.copycrc:
-                    logger.info('CRCs match for track %d', number)
+                    if self.options.keep_going:
+                        logger.warning("track %d failed to rip. "
+                                       "Continuing to next track" % number)
+                        logger.debug("adding %s to skipped_tracks"
+                                     % trackResult.filename)
+                        self.skipped_tracks.append(trackResult.filename)
+                        logger.debug(f"skipped_tracks = {self.skipped_tracks}")
```

Comment:
As whipper still seems to be compatible with Python 3.5, I would avoid using an f string here (which requires at least Python 3.6).

Reply:
I switched the f-string to a printf style format string in the latest commit.